CHANGELOG

```diff
@@ -0,0 +1,127 @@
+==== 2.1 beta 2017-12-22 ====
+After two years of hard work, we are pleased to announce Mixxx 2.1 beta. We
+have overhauled the effects system, redesigned the skins, added and improved
+lots of controller mappings, rewrote the audio file decoders twice, and of
+course fixed a bunch of bugs. Download it!
+
+Here is a quick summary of what is new in Mixxx 2.1:
+  * Graphical interface scales for high resolution screens
+  * Overhauled Deere and LateNight skins
+  * New Tango skin
+  * Effects are synchronized to the tempo
+  * Effects are processed post-fader and post-crossfader and can be previewed
+in headphones
+  * One metaknob per effect with customizable parameter control for intuitive
+use of effect chains
+  * Nine new effects: Autopan, Biquad Equalizer, Biquad Full Kill Equalizer,
+Loudness Contour, Metronome, Parametric Equalizer, Phaser, Stereo Balance,
+Tremolo
+  * Loaded effects and their parameters are saved and restored when Mixxx
+restarts
+  * More transparent sounding equalizers (Biquad Equalizer and Biquad Full Kill
+Equalizer)
+  * Improved scratching sounds with jog wheels, vinyl control, and dragging
+waveforms with the mouse
+  * Simplified looping and beatjump controls
```

Comment:
* Removed intermediate fixed point samples in the file decoding engine.
* Moved to a floating point transport engine for precise looping and beat-matching.

Reply:
While these are good improvements, I think that language will just come across as technical jargon to most users. What is the practical impact those changes make for users?

Reply:
they con notice nothing. It is only a marketing phrase.

Reply:
Okay, then I think we should leave them out of the announcement.